### ISSUE TITLE
Add Configuration to Existing Project is broken

### ISSUE THREAD

**Issue by Freyert** [GoogleCloudPlatform/cloud-code-vscode#223]
**Environment**:
*(Hint: "Report Extension Issue on Github" command will fill these out for you.)*
Extension version: 1.2.1
VSCode version: 1.43.2
OS: macOs Catalina 10.15.3

**Description**:
Trying to add configuration to an existing go project.

**Repro step**:
```
CMD + SHIFT + P to get command prompt
> Cloude Code: Run on Kubernetes
Add Configuration
```
and then I don't get the configuration wizard as described in the documentation.

**Comment by sujit-kamireddy** [comment]
Hi @Freyert ,
Does your workspace have kubernetes manifests (yaml files)?

**Comment by nnennajohn** [comment]
@Freyert , I ran into the same problem. It worked after I did `gcloud components install skaffold` and `gcloud components install skaffold`. @sujit-kamireddy I was following your voting example video from Next. I am wondering if it worked for you as you likely already had those installed. Or maybe its some other issue. Not sure.

**Comment by sujit-kamireddy** [comment]
Thanks @nnennajohn .
I think it could be one of the below two issues.
-> The workspace does not have any kubernetes manifests.
-> Necessary dependencies are not installed.

We are improving the messaging around both the above issues in the upcoming release.

**Comment by kschaab** [comment]
Fixed in v1.3.0

**Comment by nnennajohn** [comment]
Awesome. Thanks so much.
